src/Currency.php

```diff
@@ -40,6 +40,9 @@ class Currency implements Arrayable, JsonSerializable
     /** How many decimals of the currency's values should get rounded. */
     protected int $rounding;
 
+    /** Should trailing decimal zeros be trimmed. */
```

Comment:
```suggestion
    /** Should trailing decimal zeros be trimmed? */
```

Reply:
Yeah I was thinking about a question mark there, but `.`s are fine for properties. It's like "[Property indicating if] trailing decimal zeros [should] be trimmed."

Reply:
I'd normally do `.` on properties and `?` on methods. But here I used a different method docblock to make it consistent with the other methods there.
<img width="564" alt="CleanShot 2022-05-16 at 1 11 02@2x" src="https://user-images.githubusercontent.com/33033094/168580367-9e816712-e3ec-41ce-8f77-3a23c7022157.png">

Reply:
I see